src/app/pages/send/send.component.tsx

```diff
@@ -88,6 +89,20 @@ export class SendPage extends Component<IProps, IState> {
         }
     }
 
+    public async checkAddr(e) {
+        const input = e.target.value;
+        const regex = /[.](zil|eth)$/m;
```

Comment:
We avoided to check extensions till now because ENS has support for other TLDs also not only .eth. Use `this.props.blockchain` instead.

Reply:
Example of domain that should be resolved but it's not: `adrian.argent.xyz`